dist/bin/kctf-challenge

```diff
@@ -166,7 +166,7 @@ function push_image {
 
   case "${CLUSTER_TYPE}" in
     gce)
-      IMAGE_URL="${REGISTRY}/${PROJECT}/${CHALLENGE_NAME}-${IMAGE_NAME}:${IMAGE_ID}"
+      IMAGE_URL="${ARZONE}-${REGISTRY}/${PROJECT}/${CHALLENGE_NAME}/${IMAGE_NAME}:${IMAGE_ID}"
```

Comment:
Maybe let's leave this as REGISTRY and just change the registry to include the region